Explorer/Assets/Plugins/TexturesFuse/TexturesServerWrap/Unzips/ITexturesUnzip.cs

```diff
@@ -0,0 +1,185 @@
+using Cysharp.Threading.Tasks;
+using DCL.Diagnostics;
+using System;
+using System.Runtime.InteropServices;
+using System.Threading;
+using UnityEngine;
+using UnityEngine.Pool;
+using Utility.Types;
+
+namespace Plugins.TexturesFuse.TexturesServerWrap.Unzips
+{
+    /// <summary>
+    ///     Provides decoded and compressed textures from raw memory.
+    ///     Its implementations don't guarantee thread safety.
+    /// </summary>
+    public interface ITexturesUnzip : IDisposable
```

Comment:
Can I propose a better name? Something `ITexturesDecoder`? `Unzip` does not reflect the purpose at all

Reply:
I'd to avoid Decoder because it's not a decoder too... It handles also resizing, format mapping and e.t.c.

Reply:
`ITexturesProcessor` ?